app/controllers/homes_controller.rb

```diff
@@ -1,4 +1,95 @@
 class HomesController < ApplicationController
   def index
+    def consumer_key
+      OAuth::Consumer.new(
+      ENV["TWITTER_CONSUMER_KEY"],
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Indent the first parameter one step more than the start of the previous line.